### PR TITLE
Some improvements to the PacketQueue timeline.

### PR DESCRIPTION
**Pull request by JonathanLennox**

### CODE REVIEW

File: jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt [hunk 1]
```diff
@@ -1057,6 +1062,11 @@ class Endpoint @JvmOverloads constructor(
             put("rtp", rtpPacketDelayStats.toJson())
             put("rtcp", rtcpPacketDelayStats.toJson())
         }
+
+        private val timelineCounter = AtomicLong()
+        private val TIMELINE_FRACTION = 10000L
```
Comment (bgrozev) [inline jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt:1067]: Should this go in the Companion object?

Comment (JonathanLennox) [reply jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt:1067]: This is the Companion object.

Comment (bgrozev) [reply jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt:1067]: 🤦

**Review by bgrozev — APPROVED** [review]

File: jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt (hunk 1 above)
Comment (bgrozev) [reply jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt:1067]: Should the fraction be configurable?

**Pull request merged**